src/smart-components/group/role/group-roles.js

```diff
@@ -104,35 +114,59 @@ const GroupRoles = ({
           label: 'Remove from group',
           props: {
             isDisabled: !selectedRoles || !selectedRoles.length > 0,
-            variant: 'danger',
-            onClick: () => removeRoles(selectedRoles)
+            variant: 'danger'
+          },
+          onClick: () => {
+            setConfirmDelete(() => () => removeRoles(uuid, selectedRoles.map(role => role.uuid), () => fetchRolesForGroup(pagination)(uuid)));
+            setDeleteInfo({
+              title: 'Remove roles?',
+              confirmButtonLabel: selectedRoles.length > 1 ? 'Remove roles' : 'Remove role',
+              text: (selectedRoles.length > 1
+                ? <p>Members in the <b>{ `${name}` }</b> group will lose the permissions in these <b> { `${selectedRoles.length}` }</b> roles.</p>
```

Comment:
Can you make this more generic? Like a function where you can replace certain parts of text. Here is differs in `the` vs `these`  and `role` vs `roles`